grapher/scatterCharts/ScatterPlotChart.tsx

```diff
@@ -595,53 +641,16 @@ export class ScatterPlotChart
         return new Set(seriesNames)
     }
 
-    @computed private get compareEndPointsOnly() {
+    @computed get compareEndPointsOnly() {
         return !!this.manager.compareEndPointsOnly
     }
 
-    private set compareEndPointsOnly(value: boolean) {
+    set compareEndPointsOnly(value: boolean) {
```

Comment:
nit: do we need this method? This is probably fine, but I always get confused when mixing setters and @computeds with mobx. Maybe just:

```
@action.bound private onToggleEndpoints() {
       this.manager.compareEndPointsOnly = !this.compareEndPointsOnly ?? undefined
    }
```

Reply:
Good idea!